api/src/main/java/org/openmrs/module/openconceptlab/OpenConceptLabActivator.java

```diff
@@ -93,6 +97,25 @@ public void contextRefreshed() {
 		log.info("Open Concept Lab Module refreshed");
 	}
 
+	/**
+	 * If OpenMRS is unexpectedly shutdown while an OCL Import is in progress, the import record in the database
+	 * can be left in an "in progress" state, even though the import is no longer actually running.
+	 * In this situation, we want to ensure these imports are marked as failed at the next startup, so that
+	 * subsequent import attempts can succeed without errors that imports are already in progress
+	 */
+	protected void markInProgressImportsAsFailed() {
```

Comment:
Could there be a reason why this method is not private?

Reply:
OK, changed it to private @dkayiwa